packages/mediapipe-task-text/lib/src/tasks/text_classification/containers/text_classifier_result.dart

```diff
@@ -0,0 +1,58 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:mediapipe_core/mediapipe_core.dart';
+import 'package:mediapipe_text/src/third_party/mediapipe/generated/mediapipe_text_bindings.dart'
+    as bindings;
+
+/// Container with results of MediaPipe's `classifyText` task.
+///
+/// See also:
+///  * [MediaPipe's textClassifierResult documentation](https://developers.google.com/mediapipe/api/solutions/java/com/google/mediapipe/tasks/text/textclassifier/TextClassifierResult)
+class TextClassifierResult {
+  /// Generative constructor which creates a [TextClassifierResult] instance.
+  const TextClassifierResult({
+    required this.classifications,
+    this.timestamp,
+  });
+
+  /// The classification results for each head of the model.
+  final List<Classifications> classifications;
+
+  /// The optional timestamp (as a Duration) of the start of the chunk of data
+  /// corresponding to these results.
+  ///
+  /// This is only used for classification on time series (e.g. audio
```

Comment:
I may be misunderstanding, this is the text model, is the audio it is referring to here related to multimodal text classification?

Reply:
Oh this is a good point. This field is likely not necessary on `TextClassifierResult` and can be reserved for `AudioClassifierResult` and `VisionClassifierResult`.

Reply:
Looking into this more, because MP's behavior doesn't actually seem aligned with this docstring (which I copied from their Java documentation)